### DISCUSSION TITLE
Dead render of badges

### DISCUSSION THREAD

**Discussion by Eastonn** [community/community#44818] [Other Feature Feedback, Questions, & Ideas]
https://user-images.githubusercontent.com/38816055/213544958-4ea55f7a-1e9f-4b3a-806d-e94ae4c66647.MOV

When you open badge and swap to another page of it, then everything blow up

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Eastonn ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
